src/theme/components.js

```diff
@@ -249,6 +249,24 @@ const components = {
           outline: '1px auto',
         },
       },
+      select: {
+        '&.select-light': {
```

Comment:
there's a much easier way than using classes and editing everything everywhere. Indeed MUI knows about the theme!
Look at how we're doing with MuiTooltip below: using a function that takes `({theme})` as a parameter. This function can return an object with the 2 properties `&:focus` and `&:hover` depending on the theme.

Reply:
Okay. This works too. Created a `Select.ts` file under styles and moved two properties for light and dark.

Reply:
You haven't pushed the update yet, but this sounds good to me :) Carla will review it once you'll push.
Just make sure to revert all the changes that add the classes everywhere :-)

Reply:
Ah, yes that's so true, lol. I don't know why I keep going back to the longer method. Thanks @julienw.